README.md

```diff
@@ -1,42 +1,59 @@
-# ThreePio
+![open mined-logo](https://github.com/OpenMined/design-assets/blob/master/logos/OM/horizontal-primary-trans.png)
 
-[![build status](https://img.shields.io/travis/com/OpenMined/threepio.svg)](https://travis-ci.com/OpenMined/threepio)
-[![code coverage](https://img.shields.io/codecov/c/github/OpenMined/threepio.svg)](https://codecov.io/gh/OpenMined/threepio)
-[![code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/sindresorhus/xo)
-[![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
-[![made with lass](https://img.shields.io/badge/made_with-lass-95CC28.svg)](https://lass.js.org)
-[![license](https://img.shields.io/github/license/OpenMined/threepio.svg)](https://github.com/openmined/threepio/blog/master/LICENSE)
-[![npm downloads](https://img.shields.io/npm/dt/threepio.svg)](https://npm.im/threepio)
+![Tests](https://img.shields.io/github/workflow/status/OpenMined/threepio/Tests)
+![Coverage](https://img.shields.io/codecov/c/github/OpenMined/threepio)
+![License](https://img.shields.io/github/license/OpenMined/threepio)
+![OpenCollective](https://img.shields.io/opencollective/all/openmined)
 
-A multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js
+# Threepio
 
+Threepio is a multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js 
```

Comment:
I love this description and the emoji list below. But tell me some **features** of your library, too (in emoji form). You can do it below this sentence and above the three main components section. Some ideas would be: bi-directional translation, argument and keyword argument reordering, multi-lingual, works between 3 frameworks, X% coverage for Y framework, support for fuzzy finding name matching (in progress), support for versioning (in progress), support for argument typecasting (in progress). Just some ideas!